src/network/udp.py

```diff
@@ -79,7 +82,8 @@ def bm_command_addr(self):
             decodedIP = protocol.checkIPAddress(str(ip))
             if stream not in state.streamsInWhichIAmParticipating:
                 continue
-            if (seenTime < time.time() - self.maxTimeOffset or seenTime > time.time() + self.maxTimeOffset):
+            if (seenTime < time.time() - self.maxTimeOffset
+                    or seenTime > time.time() + self.maxTimeOffset):
```

Comment:
Can you fix this one please?

Reply:
This is proper formatting. The fix is in #1539